OpenRA.Mods.RA/Traits/PortableChrono.cs

```diff
@@ -107,7 +109,7 @@ public bool CanTeleport
 
 		public float GetValue()
 		{
-			return (float)((chargeTick - (25 * Info.ChargeTime)) * (Info.ChargeTime * -0.0001));
+			return (float)((chargeTick - Info.ChargeDelay) * (1f / -Info.ChargeDelay));
```

Comment:
This can simplify this down to `(float)(Info.ChargeDelay - chargeTick) / Info.ChargeDelay;`